crates/ide_completion/src/context.rs

```diff
@@ -337,25 +337,25 @@ impl<'a> CompletionContext<'a> {
                     },
                     ast::RecordExprFieldList(_it) => {
                         cov_mark::hit!(expected_type_struct_field_without_leading_char);
```

Comment:
For the record, this is one of the cases where cov_mark was exactly the thing I needed to quickly grasp the context of this code!